src/main.js

```diff
@@ -0,0 +1,242 @@
+function openNav() {
+  document.getElementById('myNav').style.display = 'block';
+}
+
+function closeNav() {
+  document.getElementById('myNav').style.display = 'none';
+}
+
+document.querySelector('.toggle').addEventListener('click', openNav);
+document.querySelector('.closebtn').addEventListener('click', closeNav);
+document.querySelectorAll('.pop-nav').forEach((link) => {
+  link.addEventListener('click', () => {
+    document.getElementById('myNav').style.display = 'none';
+  });
+});
+
+/* const projectname = document.querySelector('.projectname'); */
+const workSections = document.querySelector('.project-section');
+const modal = document.querySelector('.modal-example');
+/* const modalImg = document.querySelector('.modal-img');
+const company = document.querySelector('.company');
+const role = document.querySelector('.role');
+const year = document.querySelector('.year'); */
```

Comment:
- [ ] [OPTIONAL] Leaving unused codes as inline comments in your code makes the base messy and somewhat difficulty to read. Kindly remove these comments to make your work look professional and clean. :muscle: